x-pack/legacy/plugins/actions/np_ready/public/plugin.ts

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { unmountComponentAtNode } from 'react-dom';
+import { PluginInitializerContext, CoreSetup, CoreStart, Plugin } from 'src/core/public';
+import { i18n } from '@kbn/i18n';
+import routes from 'ui/routes';
+
+import template from '../../public/index.html';
+import { renderReact } from './application';
+import { BASE_PATH } from './application/constants';
+
+export type Setup = void;
+export type Start = void;
+
+export class ActionsPlugin implements Plugin<any, any> {
```

Comment:
You can use `Plugin`'s generics to type your setup and start return types:
```suggestion
export class ActionsPlugin implements Plugin<Setup, Start> {
```